packages/code/src/CopyButton/CopyButton.tsx

```diff
@@ -109,11 +113,13 @@ function CopyButton({ onCopy, contents }: CopyProps) {
 
   return (
     <Tooltip
```

Comment:
It seems the 100ms delay on the Tooltip hover is no longer applied. (Same for the NumberInput tooltips)